src/lib/sidebar/pipeline_forms/TimescaleSubform.svelte

```diff
@@ -0,0 +1,89 @@
+<script lang="ts">
+  import { NumberInput, Radio } from "lib/govuk";
+  import type { PipelineTimescale } from "types";
+
+  export let timescale: PipelineTimescale;
+  export let maxTimescaleFromInterventions: string | undefined = undefined;
```

Comment:
Do we have to plumb this around? If `forIntervention` is false and we're in Scheme details, we can call a helper function that looks at $gjSchemeCollection